pandas/core/window.py

```diff
@@ -879,13 +879,58 @@ def f(arg, *args, **kwargs):
                            ddof=ddof, **kwargs)
 
     _shared_docs['var'] = dedent("""
-    %(name)s variance
+    Calculate unbiased %(name)s variance.
+
+    Normalized by N-1 by default. This can be changed using the ddof argument.
 
     Parameters
     ----------
     ddof : int, default 1
         Delta Degrees of Freedom.  The divisor used in calculations
-        is ``N - ddof``, where ``N`` represents the number of elements.""")
+        is ``N - ddof``, where ``N`` represents the number of elements.
+    args
```

Comment:
I don't think the underlying function takes any additional arguments. Can remove (I think).

Reply:
I can remove it, it looks like it passes them through, but I didn't dive into it's usage.. I'll remove for now.